pennylane_lightning_gpu/src/bindings/Bindings.cpp

```diff
@@ -85,6 +88,38 @@ void StateVectorCudaManaged_class_bindings(py::module &m) {
             return new StateVectorCudaManaged<PrecisionT>(
                 data_ptr, static_cast<std::size_t>(arr.size()));
         }))
+        .def(
+            "setZeroState",
+            [](StateVectorCudaManaged<PrecisionT> &sv, const size_t index,
+               const bool use_async) {
+                std::complex<PrecisionT> value(0, 0);
```

Comment:
Is this to create |0...0> state?
Shouldn't the complex value be `(1,0)` then, and applied at the correct index?

Reply:
No, it is used to set the  |0...0> state to be (0,0) (since |0...0> state is set to be (0,1) when constructed) and then we use _create_basis_state_GPU to set the target state.

Reply:
So this sets the value at a given index to (0,0)?
If so, I think we should rename this, as the name implies setting a |0...0> state.

Though, I think I'd prefer if we adapt the constructor to make setting the `(1,0)` value optional, which would remove the need for this. What do you think?

Reply:
sure! Thanks

Reply:
done